.coveragerc

```diff
@@ -430,7 +430,7 @@ omit =
     homeassistant/components/kaiterra/*
     homeassistant/components/kankun/switch.py
     homeassistant/components/keba/*
-    homeassistant/components/keenetic_ndms2/device_tracker.py
+    homeassistant/components/keenetic_ndms2/*
```

Comment:
This excludes the config flow from tests, which it should not.
The configuration flow requires tests with 100% coverage.

Reply:
Got it. Fixing this